### DISCUSSION TITLE
Time profile analysis predicted line

### DISCUSSION THREAD

**Discussion by safgren** [Open-Systems-Pharmacology/Forum#1422] [Questions & Problems] [answered]
Hi, I am a new user learning the software.   I have a observed data for a 1 week interval that I included on the time profile.  The predicted curve stops at 24 hours and I would like to extend this to the full week.  I have tried to adjust the administration protocol with no luck.  I looked for a way to modify the graph with the chart editor but don't see an option for this.  Please help.  It is probably an easy fix but I don't see it.  Thanks.

**Comment by StephanSchaller** [comment, marked as answer]
Dear @safgren, you have to increase the simulation time to 7 days:

![image](https://user-images.githubusercontent.com/25477643/210282985-f6251277-bef1-44b1-9d09-bc746425cb0d.png)

**Reply by safgren** [reply]
Thank you!
